internal/proxyservice/tcp/mssql/connector.go

```diff
@@ -0,0 +1,21 @@
+package mssql
+
+import (
+	"net"
+
+	"github.com/cyberark/secretless-broker/pkg/secretless/log"
+	"github.com/cyberark/secretless-broker/pkg/secretless/plugin/connector"
+)
+
+type SingleUseConnector struct {
```

Comment:
can we add some comments here? what is this struct's purpose? which parameters will it probably have?